### PR TITLE
Release/1.4.0

### PR DESCRIPTION
**Pull request by ThinkTankShark**
Upgrades dev dependencies to latest version.

**Fixes Issues:**
-  #6 
- #14
- #15
- #16
- #25

**Related PR:**
- #35 
- #36 
- #37 
- #38 
- #39 
- #40 
- #41 
- #42 
- #43 
- #44 
- #45

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,30 +64,33 @@
...
+    "istanbul-lib-instrument": "^1.9.1",
+    "jest": "^22.0.4",
+    "jest-cli": "^22.0.4",
+    "lint-staged": "^6.0.0",
     "strict-loader": "^1.1.0",
-    "webpack": "^2.2.1"
+    "webpack": "^3.10.0"
+  },
+  "dependencies": {
+    "trevor": "^2.3.0"
```
Comment (isuttell) [inline package.json]: This should be a dev dependency. We don't need to install this in all of the apps.

File: package.json [hunk 2]
```diff
@@ -9,8 +9,8 @@
     "lint": "./node_modules/eslint/bin/eslint.js src/**/*.js",
     "test": "npm run lint && ./node_modules/jest-cli/bin/jest.js --coverage",
     "watch:test": "./node_modules/jest-cli/bin/jest.js --coverage --watch",
-    "coverage": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js -v",
-    "travis": "npm install -g trevor && trevor"
+    "coverage": "./node_modules/jest-cli/bin/jest.js --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js -v",
```
Comment (isuttell) [inline package.json]: We're running the unit tests twice. Why? We've already generated the code coverage with the test command.

Comment (ThinkTankShark) [reply package.json]: Fixed.

File: package.json (hunk 1 above)
Comment (ThinkTankShark) [reply package.json]: Fixed.

**Comment by ThinkTankShark** [comment]
@isuttell plz review again. tnx

**Pull request merged**